.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  "recommendations": [
+    "astro-build.astro-vscode",
```

Comment:
This is way too opinionated to start with. Can we keep this to just `"astro-build.astro-vscode"` for now?

Reply:
Ok, and what about the root `.vscode`, that will only be used contributors? Maybe something like this:

```json
{
  "recommendations": [
    "astro-build.astro-vscode",
    "esbenp.prettier-vscode",
    "editorconfig.editorconfig",
    "dbaeumer.vscode-eslint"
  ],
  "unwantedRecommendations": []
}
```

Reply:
If Nate's happy I'm happy! Thanks for making this change @JuanM04 !